tests-perf/locust/locust-notifications.py

```diff
@@ -12,13 +12,50 @@ class NotifyApiUser(HttpUser):
     wait_time = constant_pacing(60)
     host = "https://api.staging.notification.cdssandbox.xyz"
 
-    @task
-    def send_notifications(self):
-        headers = {"Authorization": AUTH_HEADER}
-        json = {
+    def __init__(self, *args, **kwargs):
+        super(NotifyApiUser, self).__init__(*args, **kwargs)
+
+        self.headers = {"Authorization": AUTH_HEADER}
+        self.json = {
             "email_address": "success@simulator.amazonses.com",
             "template_id": "9c17633c-126a-4ad3-ad2f-b14c3a85314a",
             "personalisation": {"colour": "Fulvous"},
         }
 
-        self.client.post("/v2/notifications/email", json=json, headers=headers)
+    @task
+    def task(self):
+        pass
+
+class OneOffEmailMessage(NotifyApiUser):
+    @tag("one-off-email")
+    @task
+    def send_one_off_email_notifications(self):
+        self.client.post("/v2/notifications/email", json=self.json, headers=self.headers)
+
+class OneOffEmailWithFileAttachmentMessage(NotifyApiUser):
+    @tag("one-off-email-attached")
+    @task
+    def send_one_off_email_with_attachment_notifications(self):
+        self.json["personalisation"] = {
+            "application_file": {
+                "file": "Q29udGVudCBvZiBBdHRhY2hlZCBmaWxl",
+                "filename": "attached_file.txt",
+                "sending_method": "attach",
```

Comment:
Followed the schema definitions [1] and the smoke tests.. still getting 400 bad requests 

[1] https://github.com/cds-snc/notification-api/blob/master/app/schema_validation/definitions.py#L22-L42

Reply:
That was defined in error and part of debugging.. I remember seeing the example as part of the smoke tests.. https://github.com/cds-snc/notification-api/blob/4dcb11eb1f893b4bfa4f2142a7ff8b7b0f462d32/tests_smoke/smoke/test_api_one_off.py#L38

Reply:
@jimleroyer I have updated the templated id to `5ebee3b7-63c0-4052-a8cb-387b818df627`..

Reply:
Ok that one is on me. The services need to have a switch turned on in the admin settings to be  able to send attachments. It was disabled for this service:
![image](https://user-images.githubusercontent.com/805567/133129333-37b19f42-eb84-41ec-977a-00d2d6c1b5a2.png)

Can you try again?

Reply:
Still getting the same 400 bad request error

Reply:
I was able to make it work for me locally. I changed the code a bit to get rid of `self.json` and inline explicitly each content message. I also observed that there is an inexistent `var` used somewhere that would need to be switched to `application_file`.